com.github.TuxPusher.yaml

```diff
@@ -0,0 +1,27 @@
+app-id: com.github.mrbid.TuxPusher
```

Comment:
also the manifest file name should match the app-id.

Reply:
woops, updated